generators/Generators.fs

```diff
@@ -1672,4 +1672,29 @@ type Zipper() =
         [tree; sut; expected]
 
     override __.TestMethodName canonicalDataCase = 
-        base.TestMethodName canonicalDataCase |> String.replace "Set_" "Set "
\ No newline at end of file
+        base.TestMethodName canonicalDataCase |> String.replace "Set_" "Set "
+
+type RestApi() = 
+    inherit GeneratorExercise()
+
+    override this.PropertiesWithIdentifier canonicalDataCase = this.Properties canonicalDataCase
+
+    override __.RenderInput (canonicalDataCase, key, value) =
+        match key with
+        | "database" -> "\"\"\"" + value.ToString(Newtonsoft.Json.Formatting.None) + "\"\"\""
+        | "payload" -> "\"\"\"" + value.ToString(Newtonsoft.Json.Formatting.None) + "\"\"\""
+        | _ -> base.RenderInput (canonicalDataCase, key, value)
+
+    override __.RenderExpected (canonicalDataCase, key, value) =
+        match key with
+        | "expected" -> ("\"\"\"" + value.ToString(Newtonsoft.Json.Formatting.None) + "\"\"\"")
+        | _ -> base.RenderExpected (canonicalDataCase, key, value)
+
+    override __.RenderArrange canonicalDataCase =
+        base.RenderArrange(canonicalDataCase) @ ["let api = RestApi(database)"]
+
+    override this.RenderAssert canonicalDataCase = 
```

Comment:
I think that perhaps you could simplify this a bit by overriding the `RenderSut` method instead of the `RenderAssert` method. Could you check to see if that is correct?